app/views/delivery_zones/index.html.erb

```diff
@@ -0,0 +1,19 @@
+<%= render "application/admin_header" %>
+
+<div class="content-wrapper-narrow">
+  <section class="zipcode-group">
+    <label for="">Active Zipcodes</label>
```

Comment:
@corwinharrell is this semantically correct?

To me, this feels more like an `<h2>` than a `<label>`, since `<label>` elements reference form inputs.

What do you think?